inspire_disambiguation/core/helpers.py

```diff
@@ -320,6 +321,16 @@ def process_clustering_output(clusterer):
     return output
 
 
+def train_validation_split(curated_signatures_dict, split_fraction):
+    seed(99)
+    train_signatures_uuids = sample(list([*curated_signatures_dict]),
+                                    k=int(len([*curated_signatures_dict]) * split_fraction))
+    train_signatures = [curated_signatures_dict[uuid] for uuid in train_signatures_uuids]
+    test_signatures_uuids = set(list(curated_signatures_dict)) - set(list(curated_signatures_dict))
```

Comment:
This is the empty set. I think you mean
```suggestion
    test_signatures_uuids = set(curated_signatures_dict.keys()) - train_signatures_uuids
```